catalog/core/catalog-core-versioning/versioning-common/src/main/java/ddf/catalog/core/versioning/impl/DeletedMetacardImpl.java

```diff
@@ -70,20 +65,17 @@
         METACARD_TYPE = new MetacardTypeImpl(PREFIX, DESCRIPTORS);
     }
 
-    public DeletedMetacardImpl(String deletionOfId, String deletedBy, String lastVersionId,
-            Metacard metacardBeingDeleted) {
-        super(metacardBeingDeleted,
-                new MetacardTypeImpl(PREFIX,
-                        getDeletedMetacardType(),
-                        metacardBeingDeleted.getMetacardType()
-                                .getAttributeDescriptors()));
+    public DeletedMetacardImpl(String id, String deletionOfId, String deletedBy,
+            String lastVersionId, Metacard metacardBeingDeleted) {
+        super(metacardBeingDeleted, new MetacardTypeImpl(PREFIX,
+                getDeletedMetacardType(),
+                metacardBeingDeleted.getMetacardType()
+                        .getAttributeDescriptors()));
```

Comment:
Hmmm... My IntelliJ formats it like it was previously. I just pulled ddf-support and re-imported formatters and it looks like it does it the same still. Your formatter might be out of date.
```java
public DeletedMetacardImpl(String id, String deletionOfId, String deletedBy,
            String lastVersionId, Metacard metacardBeingDeleted) {
        super(metacardBeingDeleted,
                new MetacardTypeImpl(PREFIX,
                        getDeletedMetacardType(),
                        metacardBeingDeleted.getMetacardType()
                                .getAttributeDescriptors()));
        this.setDeletionOfId(deletionOfId);
        this.setDeletedBy(deletedBy);
        this.setLastVersionId(lastVersionId);
        this.setTags(ImmutableSet.of(DELETED_TAG));
        this.setId(id);
    }```